### ISSUE TITLE
Update INSTALL instructions

### ISSUE THREAD

**Issue by stylewarning** [fricas/fricas#41]
The [INSTALL](https://github.com/fricas/fricas/blob/master/INSTALL) instructions are rather stale and could use house cleaning. For instance, it refers to SVN, whereas development has moved to GitHub. In my personal optinion, the instructions could also be moved to Markdown for improved readability and nice sectioning.

**Comment by oldk1331** [comment]
The `INSTALL` file can be and should be updated, with newer and tested information.

As for Markdown, I guess Waldek would oppose.  Current text file does not look bad.

**Comment by stylewarning** [comment]
Markdown is pretty text-friendly. Maybe I'll take a stab at updating it, and I won't be offended if the maintainers don't like the direction I go. :)

**Comment by hemmecke** [comment]
Hmmm... Markdown? That would somehow go against my attempt to also show the installation instructions at https://fricas.github.io. I have a work-in-progress branch with already translations of README, FAQ and INSTALL to .rst format.
https://github.com/hemmecke/fricas/tree/wip/rstdoc/src/doc/sphinx/source
Unfortunately, I haven't worked on it for quite some time, but intend to integrate that documentation to the FriCAS repo, when I am satisfied with it.

**Comment by hebisch** [comment]
On Fri, Mar 05, 2021 at 04:17:58PM -0800, Robert Smith wrote:
> The [INSTALL](https://github.com/fricas/fricas/blob/master/INSTALL) instructions are rather stale and could use house cleaning. For instance, it refers to SVN, whereas development has moved to GitHub. In my personal optinion, the instructions could also be moved to Markdown for improved readability and nice sectioning.

INSTALL corresponds to state at previous release, which were from svn.
Incoming release will be from git and of course INSTALL will be
updated.  Yes, it would be better to update it at time of switch...

INSTALL should be read using plain text viewer and clearly plain
text format is most readible in plain text viewer.  IMO
marginal estetic gains of markup do not compensate risk of
mangling important information. 

-- 
                              Waldek Hebisch

**Comment by hemmecke** [comment]
@hebisch  Have you looked at
https://github.com/hemmecke/fricas/blob/wip/rstdoc/src/doc/sphinx/source/install.rst ?
I have already done some rewriting. I think it would be a step back to insist on a plain text INSTALL and do the editing again. 
I don't see a "risk of mangling important information", rather more visibility, if I can nicely integrate it at https://fricas.github.io embellished with hyperlinks.

**Comment by stylewarning** [comment]
I took a stab at at least refactoring the current `INSTALL` file in this PR: https://github.com/fricas/fricas/pull/45

If it happens to get merged, I won't mind if it gets wholesale replaced by e.g. @hemmecke's work.

I would also note that Markdown is a plain-text format and is very easily readable in any plaintext editor or viewer. With that said, I decided to keep the spirit of "true" plaintext in my PR.
